content/news/2018-11-16-the-navcoin-community-fund-is-live.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: news
+generated_by_cms: true
+title: The NavCoin Community Fund is live
+author: Kieren Hyland
+date: '2018-11-16T11:00:18+13:00'
+feature_image: /images/uploads/community-fund-live.png
+categories:
+  - Nav News
+---
+The latest updates to the NavCoin Core protocol have activated, and as a result, the NavCoin Community Fund is ready to accept your proposal for consideration and funding by the network. 
+
+The latest updates to the NavCoin Core protocol have activated, and as a result, the NavCoin Community Fund is ready to accept your proposal for consideration and funding by the network. 
+
+### A world first!
+
+Delivering a major upgrade like the Community Fund is a significant achievement - both for the Core dev team and all the other community members who contributed their time and skill to bring it to life. 
+
+In case you aren’t already aware, the NavCoin Community fund is the world’s first and only fully-decentralised fund. Both in the way it’s governed and how the funds are managed. 
```

Comment:
we could link to one of our articles perhaps? good for seo and will inform people